### DISCUSSION TITLE
Can you rename Kazakhstan's capital to Astana?

### DISCUSSION THREAD

**Discussion by nomadines** [breezy-weather/breezy-weather#1406] [Ideas]
### Describe your idea

I know that it is minor issue but the name provided by Breezy is a reminder of very tragic events for my countrymen and me personally. I would really appreciate it. 

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
There are no city names in Breezy Weather: the data is provided by weather sources. I can find "Astana" using the default search source, so you can either change your source or contact the source providing the old name for your city and let them know about the issue.
